cypress/tests/core/blocks-listing-templates.js

```diff
@@ -48,7 +48,7 @@ describe('Folder Contents Tests', () => {
     cy.url().should('eq', Cypress.config().baseUrl + '/my-folder/my-document');
     cy.get('.listing-item img')
       .should('have.attr', 'src')
-      .and('contain', '/my-folder/my-document/my-image/@@images/image/preview');
+      .and('contain', '/my-folder/my-document/my-image/@@images/image/icon');
```

Comment:
@nzambello Maybe we should use thumb or something else as there are discussions to remove low resolution scales. @tisto ^